data/mods/gen4/items.js

```diff
@@ -125,17 +125,6 @@ let BattleItems = {
 		onEffectiveness() {},
 		desc: "Holder's Speed is halved and it becomes grounded.",
 	},
-	"jabocaberry": {
```

Comment:
@Marty-D you fixed these in edc2b8f98a08477c858

But according to #6346, all damaging-hit handlers are timed like these, so this is no longer necessary, right?